src/modules/cities/services/data-finder.ts

```diff
@@ -61,13 +61,19 @@ const createCity = ({ city, totalCases, state }: CityCsvRow): City => {
     longitude
   }
 }
+const treatData = (cityRow: CityCsvRow): boolean => {
+  const name = toCityName(cityRow.city)
+  return !['NÃO ESPECIFICADA', 'NAO ESPECIFICADA'].includes(name.toUpperCase())
```

Comment:
Como @vmarcosp sugeriu, voce colocar os valores em uma constante e tratar como regex, pensei em algo como:

```suggestion
  const INVALID_REGEX = /n[aã]o\s+especificada/i

  return INVALID_REGEX.test(name)
```

Mas é só minha sugestão, agora é contigo :D

Reply:
boa ideia :)